src/server/eval/unary_operation_node.cpp

```diff
@@ -36,7 +36,7 @@ UnaryOperationNode::UnaryOperationNode(op::UnaryOperationEnum operation, const n
 string UnaryOperationNode::eval(GeoObject* obj) const
 {
 	// This is needed beacase of MSVC
-	assert(false);
```

Comment:
assert if OK, but VS2015 compiler says "no return in function eval()" and gives an error.